docs/application/tizen-studio/TOC.md

```diff
@@ -43,45 +44,45 @@
 #### [File Analysis](common-tools/da-file.md)
 #### [Network Analysis](common-tools/da-network.md)
 #### [OpenGL Analysis](common-tools/da-opengl.md)
-#### [UI Hierarchy Analysis](common-tools/da-ui-hierarchy-n.md)
-### [Call Stack View](native-tools/call-stack-view-n.md)
-### [Static Analyzer](native-tools/static-analyzer-n.md)
-### [Valgrind](native-tools/valgrind-n.md)
-### [T-trace](native-tools/t-trace-n.md)
-### [Address Sanitizer](native-tools/address-sanitizer-n.md)
-### [Code Coverage](native-tools/code-coverage-n.md)
+#### [UI Hierarchy Analysis](common-tools/da-ui-hierarchy.md)
```

Comment:
Current file name is da-ui-hierarchy-n.md

Reply:
I renamed the file name.